README.md

```diff
@@ -113,6 +113,10 @@ declare global {
 }
 ```
 
+### Upgrading from version 6 -> 7
```

Comment:
Latest version on NPM is 7.1.0 https://www.npmjs.com/package/@brightlayer-ui/react-native-components?activeTab=versions.

So it should be 7 -> 8